### DISCUSSION TITLE
Change searchnode logstash heap size

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#1780] [Unsupported Versions] [answered]
Hello,
I'm using SO 2.3 and trying to edit logstash heap size, in minion configurations (/opt/so/saltstack/local/pillar/minions/****_searchnode.sls).
I edited lsheap value in minion sls file, and restarted searchnode logstash, when comparing ram before and after, nothing had changed.
In logstash docker when viewing 'ps aux' i can see that java started with default heap size value:
`/bin/java -Xms1g -Xmx1g ...`
And when viewing the java heap environment variables in logatash docker they seems ok:
`LS_JAVA_OPTS=-Xms20g -Xmx20g`

**Comment by weslambert** [comment, marked as answer]
This value (`lsheap`) can be changed in `/opt/so/saltstack/local/pillar/minions/<minion_name>` on the manager.  You can then target the minion and apply the Logstash state with something like:

salt-call <minion_name> state.apply logstash
